packages/ui-keyring/src/types.d.ts

```diff
@@ -35,6 +40,7 @@ export type State = {
 };
 
 export type KeyringInstance = {
+  changeAccountPassword: (t: TranslationFunction, address: string, password: string, newPassword: string) => AccountResponse,
```

Comment:
No. We cannot do this. We should not be passing `t` around between functions, it is specific to a component and internally yo it only, nowhere else.